app/build.gradle

```diff
@@ -34,25 +34,27 @@ android {
                     'proguard-rules-retrofit.pro'
         }
     }
+    dynamicFeatures = [':paintings']
 }
 
 dependencies {
-    implementation libraries.androidArchLifecycleExtensions
+    api libraries.androidArchLifecycleExtensions
```

Comment:
since the base feature module will always be installed on device, it makes sense for this one to contain dependencies that will be used across the app.

marking these dependencies as `api` means that any feature module that depends on `app` can also use these dependencies.